scripts/libbluray/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+LIBLURAY_ADDITIONAL_FLAGS=
+
+CC=${FAM_CC} \
+AR=${FAM_AR} \
+AS=${X264_AS} \
```

Comment:
Please don't use variables made by other scripts. In case the libx264 isn't listed to be built, this variable will be empty.
Evaluate the AS path on your own in this script.

Reply:
ok